Qpf/Macro/NameUtils.lean

```diff
@@ -0,0 +1,17 @@
+open Lean Meta
+
+namespace Lean.Name
+-- This function has diffreent behaviour from Name.replacePrefix
+def replacePrefix2 (old_pref new_pref : Name) : Name → Name
```

Comment:
Please explain *what* the difference is.
Also, I see you've removed the original `replacePrefix` anyway, so it's probably fine to remove this comment (but explain the difference in the PR description!) and rename it to `replacePrefix`. Otherwise, you should rename it to something that at least hints to what the difference with `replacePrefix` is.

Reply:
@alexkeizer I can't really tell what the difference is as this is the one you had written, tho lean has a function called `Name.replacePrefix` that previously was shadowed.

Reply:
Fixed now